composer.json

```diff
@@ -22,9 +22,9 @@
         "amphp/parallel-functions": "1.0",
```

Comment:
Maybe best to upgrade amp/paralllel-functions to 1.1 here?
Otherwise opis will still be used ?

Reply:
I had some issues running the tests locally, had the same errors as in the pipelines on my local machine, even on master. So I assumed something was wrong with my machine.

I can try upping `amphp/parallel-functions` to 1.1, but then we're back on the `The given data could not be serialized` error :confused:

Reply:
Yes, that's an error we need to get resolved in  this PR. It should work for parallel functions ^1.1

Some of the pipelines were indeed failing earlier today as well.
but in order to get this released in a trustworthy way, all actions should be green again